pkg/operator/configobservation/images/observe_images.go

```diff
@@ -169,15 +163,6 @@ func ObserveAllowedRegistriesForImport(genericListers configobserver.Listers, re
 			return prevObservedConfig, append(errs, err)
 		}
 	}
-
-	existingLocations := []configv1.RegistryLocation{}
-	for _, location := range existingAllowedRegistries {
-		existingLocations = append(existingLocations, location.(configv1.RegistryLocation))
-	}
-	if !reflect.DeepEqual(existingLocations, configImage.Spec.AllowedRegistriesForImport) {
```

Comment:
@bparees why was this ripped off? afaik this part worked fine?

Reply:
It did not work fine:
https://github.com/openshift/cluster-kube-apiserver-operator/pull/188#issuecomment-454184145

483 events from this, when the array was empty.